style-dictionary/build.js

```diff
@@ -5,7 +5,7 @@ function build(source, destination) {
       scss: {
         transformGroup: 'scss',
         transforms: ['name/css_composite', 'value/gradient', 'value/shadow'],
-        buildPath: '../style-dictionary-dist/',
+        buildPath: '../src/style-dictionary-dist/',
```

Comment:
@constanzauanini i moved this into src/ directory because we need this packaged into the distribution